pypeit/tests/test_telluric.py

```diff
@@ -8,6 +8,8 @@
 from pypeit.tests.tstutils import telluric_required, tell_test_grid
 from pypeit import data
 
+#TODO -- Consider moving this into the DevSuite tesets
+
```

Comment:
I think anything that requires the telluric files should be moved the dev-suite.

Reply:
Agreed, I moved this to the dev-suite